### DISCUSSION TITLE
Team back from holiday !!

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#108] [Announcements]
Team is back we will continue working on JoomCCK, In next update we will release extended version (will not be free) and free version will be here in repo.

Also we will start on apps mode feature , and implementation of AI.

Thanks for everyone helping us on testing and support.

regards,
ANDI

**Comment by Giorgi625** [comment]
Great, I also got some free time now and will get back to testing. Hope we will be able to release stable version soon.

**Reply by andiamine** [reply]
Thanks :)

**Comment by Giorgi625** [comment]
Also I was not able to add some suggestions to Working hour fields as it was moved to new features page.
I think best option will be to create something like Google maps working hours, where it also shows if its open currently or closed
and it should have option to use 24 hours also, because in many countries people do not use AM/PM time system.

![image](https://github.com/JoomCoder-com/JoomCCK/assets/16633699/0113ba05-2292-4009-8ff0-69b579842b8a)

**Reply by andiamine** [reply]
We will add field that does this :)
